kuksa-client/requirements.txt

```diff
@@ -12,15 +12,15 @@ colorama==0.4.6
     # via cmd2
 decorator==5.1.1
     # via jsonpath-ng
-grpcio==1.56.2
+grpcio==1.63.0
```

Comment:
Had to update the version because of the following message:

```The grpc package installed is at version 1.56.2, but the generated code in kuksa/val/v1/val_pb2_grpc.py depends on grpcio>=1.63.0. Please upgrade your grpc module to grpcio>=1.63.0 or downgrade your generated code using grpcio-tools<=1.56.2. This warning will become an error in 1.65.0, scheduled for release on June 25, 2024.```

which then needed a new version of protobuf to be compatible

Reply:
I think this version should also be updated in pyproject.toml and or setup.cfg  This file is generated, see comment on top. Maybe @erikbosch  remembers which is the "source of truth

Reply:
Oversight on my side. Thanks for the note. I updated both of them :)

Reply:
We have a description for updating dependencies at release at: https://github.com/eclipse-kuksa/kuksa-python-sdk/wiki/Release-Process#update-dependencies

```
cd kuksa-client
pip-compile --upgrade --resolver=backtracking setup.cfg
pip-compile --upgrade --extra=test --output-file=test-requirements.txt --resolver=backtracking setup.cfg
```

If you do like that you even get a few more dependencies updated.

Reply:
Done - I did a short test again and everything looked fine, however I did only check the basic functionality (subscribe, get, set with grpc) and not wss stuff or other things